packages/patternfly-4/react-core/src/components/Dropdown/Dropdown.d.ts

```diff
@@ -1,16 +1,7 @@
 import { SFC, HTMLProps, ReactType, ReactNode } from 'react';
 import { OneOf } from '../../typeUtils';
 import { DropdownToggleProps } from './DropdownToggle';
-
-export const DropdownPosition: {
-  right: 'right';
-  left: 'left';
-}
-
-export const DropdownDirection: {
-  up: 'up';
-  down: 'down';
-}
+import { DropdownPosition, DropdownDirection } from './dropdownConstants';
```

Comment:
Can the file name start uppercase?

Reply:
looks like out constants file in the internal directory start with lower case.